dts/arm/st/l1/stm32l1.dtsi

```diff
@@ -200,6 +200,16 @@
 			status = "disabled";
 		};
 
+    spi3: spi@40003c00 {
```

Comment:
SPI3 is not available on the complete STM32L1x series but only on STM32L1xxxC/D/E.
So this definition should be added in stm32l151Xc.dtsi, stm32l152Xc.dtsi and stm32l152Xe.dtsi (like it is done for timers5 for example)

Reply:
Done. Thanks for the suggestion.